ykman-gui/py/yubikey.py

```diff
@@ -50,21 +63,13 @@ def wrapped(*args, **kwargs):
         try:
             return f(*args, **kwargs)
 
-        except YkpersError as e:
-            if e.errno == 3:
-                return failure('write error')
-            if e.errno == 4:
-                return failure('timeout')
-
-            logger.error('Uncaught exception', exc_info=e)
-            return unknown_failure(e)
-
-        except FailedOpeningDeviceException:
-            return failure('open_device_failed')
-
         except smartcard.pcsc.PCSCExceptions.EstablishContextException:
             return failure('pcsc_establish_context_failed')
 
+        except ValueError as e:
```

Comment:
TODO: change to a more specific exception. Should be changed on ykman cli first.